src/components/features/rental/club/club.php

```diff
@@ -1,4 +1,4 @@
-<div class="flex justify-start items-center gap-10">
+<div class="flex phone:hidden justify-start items-center gap-10">
   <img src="<?php echo $absoluteUriPath ?>/asset/rental/club.png" />
```

Comment:
[imo]
`desktop`と`phone`にhtmlの差分がそこまで無いから`class`のやりくりだけでOK！

```suggestion
<div class="flex phone:flex-col justify-start items-center gap-10">
  <img class="phone:w-[calc(100%-32px)]" src="<?php echo $absoluteUriPath ?>/asset/rental/club.png" />
```